productivity-suite/piercing-library/src/message-bus/inline-script.ts

```diff
@@ -0,0 +1,4 @@
+import { createBrowserMessageBusFromStateStr } from "./browser-message-bus";
+
+const stateStr = "__MESSAGE_BUS_STATE_STR_PLACEHOLDER__";
```

Comment:
using a placeholder and replacing it when we apply the script was the only way I could think of to have the inline script built and apply the state when we add actually make use of the script.

alternatively I think we could also have the `stateStr` read from some global variable and update that variable before using the script...

does anyone has a better solution? 🙂

Reply:
Better solution suggested by Pete 🙂